### DISCUSSION TITLE
Sintax highlight for code blocks in markdown

### DISCUSSION THREAD

**Discussion by duartecancela** [vercel/next.js#14799] [Help] [answered]
Hi, how can i have sintaxe highlight  for code blocks in md posts?

**Comment by youling4438** [comment]
I have the same question, and I add the ``highlight.js``  to highlight for code blocks in md posts, but can't resolve highlight style theme file(``` import 'highlight.js/styles/github.css'```) in ``_app.js``

**Comment by leerob** [comment, marked as answer]
There are a few different ways:

- Use a library like Highlight.js or Prism.js
- Use MDX and `MdxProvider` to specify which React components you want to use to render the elements
- Add global CSS styles targeting those elements inside `pre` or `code` (don't recommend)

Good example: https://github.com/tailwindlabs/blog.tailwindcss.com
